docs/user/api/index.md

```diff
@@ -52,3 +52,144 @@ deprecation. Existing consumers should migrate to JSON/RSS/[Index APIs].
+
+## Integration guide
+
+Many tools already integrate with PyPI, uploading packages or
+retrieving data; see [the Python Packaging Guide's tool recommendations].
+
+### Migrating to the new PyPI
+
+Warehouse has now replaced the legacy PyPI site that was deployed at
+<https://pypi.python.org/>. If your site/service
+used to link or upload to pypi.python.org, it may continue to work due
+to redirects, but you should use <https://pypi.org> instead.
+
+You should also watch [our status page] and subscribe to
+[the PyPI blog] and
+[the PyPI announcement list (low-traffic)] to find out about future changes.
+
+Here are some tips.
+
+!!! note
+
+    `{name}` is the name of the package as represented in the URL;
+    for `https://pypi.org/project/arrow/`, you'd insert `arrow`
+    wherever you see `{name}`.
+
+* If your client correctly follows redirects, you can replace
+  `pypi.python.org` in your links with `pypi.org` and everything
+  should just work. For instance, the project detail page
+  `https://pypi.org/pypi/{name}` (with or without a trailing slash)
+  redirects to `https://pypi.org/project/{name}/`.
+
+* Shorter URL: `https://pypi.org/p/{name}/` will redirect to
+  `https://pypi.org/project/{name}/`.
+
+* All APIs: [access is HTTPS-only]
+  (changed in October 2017). Furthermore, `pypi.org` honors an
+  `Accept-Encoding: gzip` header, whereas `pypi.python.org` ignored it.
+
+* JSON API: `https://pypi.org/pypi/{name}/json` returns the
+  expected JSON response directly. See [JSON API].
+
+* XML-RPC API: See [Changes to XMLRPC API]. Will be deprecated in
+  the future (no specific end date set yet); switch to the [RSS Feeds],
+  [Index API], or [JSON API].
+
+* Packages/updates RSS feeds: `https://pypi.org/pypi?%3Aaction=rss`
+  redirects to `https://pypi.org/rss/updates.xml`, and
+  `https://pypi.org/pypi?%3Aaction=packages_rss` redirects to
+  `https://pypi.org/rss/packages.xml`. See [RSS Feeds] for
+  descriptions. The data differs from the legacy feed data because
+  the new feeds are standards-compliant and fix inaccuracies in the
+  publication date; see #3248 for more information.
+
+* Documentation upload: Users can no longer use `doc_upload` in the
+  API to upload documentation ZIP files, separate from packages, to be
+  hosted at `pythonhosted.org`. See #509 for more information.
+
+* `User-Agent` Filtering: Some client user agents were filtered to
+  always use `legacy.pypi.org`, a temporary deployment of the legacy
+  PyPI codebase, regardless of brownouts or redirects, in order to
+  give them extra time to migrate. On 30 April 2018,
+  `legacy.pypi.org` was shut down, so all clients use `pypi.org`
+  regardless of their `User-Agent`.
+
+* Subscribe to [the PyPI blog] and [the PyPI announcement list (low-traffic)].
+
+If you're a PyPI end user or packager looking to migrate to the new
+PyPI, [see the official Python Packaging User Guide on migrating to PyPI].
```

Comment:
This is pretty dated at this point (~6 years old?) I think it could be condensed or eliminated entirely, @ewdurbin thoughts?

Reply:
The linked page on packaging.python.org is marked as Obsolete... and I agree. 6+ years is long enough.

Reply:
Cool, will delete!

Reply:
Done, PTAL 🙂